pkg/proxy/util/nfacct/nfacct_linux_test.go

```diff
@@ -337,6 +338,11 @@ func TestRunner_Get(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
+			if runtime.GOARCH == "s390x" {
+				if len(tc.handler.responses) > 0 {
```

Comment:
While it may be the case that the tests _currently_ fail if and only if `responses` is set, the tests might change or be extended in the future. I would just skip all of these tests unconditionally on s390x.

Reply:
but please add a comment explaining why too (not exactly sure where the comment needs to go to not interfere with the build directive...)